File: time/src/main/java/com/tr/hsyn/time/TimeMillis.java
package com.tr.hsyn.time;


import java.util.concurrent.TimeUnit;


/**
 * Zaman birimlerinin milisaniye değerlerini tutar.
 * Bu değerler sabittir ve en kaba şekilde değerlendirilir.<br><br>
 * <p>
 * Mesela 1 gün daima 24 saattir.<br>
 * Mesela 1 hafta daima 7 gündür.<br>
 * Mesela 1 ay daima 30 gündür.<br>
 * Mesela 1 yıl daima 365 gündür.<br><br>
 *
 * <p>
 * Bu sebeple, bu değerler üzerinden ince hesap yapmak doğru değildir.
 * </p>
 */
public interface TimeMillis {

    long SECOND = 1000L;
    long MINUTE = TimeUnit.MINUTES.toMillis(1); //my favor//
    long HOUR   = TimeUnit.HOURS.toMillis(1);
    long DAY    = TimeUnit.DAYS.toMillis(1);
    long MONTH  = 30L * DAY;
    long YEAR   = 365L * DAY;
}

File: timek/src/main/java/tr/xyz/timek/TimeMillis.kt
package tr.xyz.timek

/**
 * Defines all time units in milliseconds.
 */
interface TimeMillis {
	
	companion object {
		/**
		 * Milliseconds in a second
		 */
		const val SECOND = 1000L
		/**
		 * Milliseconds in a minute
		 */
		const val MINUTE = SECOND * 60
		/**
		 * Milliseconds in an hour
		 */
		const val HOUR = MINUTE * 60
		/**
		 * Milliseconds in a day
		 */
		const val DAY = HOUR * 24
		/**
		 * Milliseconds in a month
		 */
		const val MONTH = DAY * 30
		/**
		 * Milliseconds in a year
		 */
		const val YEAR = DAY * 365
	}
}